src/i965_drv_video.h

```diff
@@ -69,7 +69,9 @@
 #define DEFAULT_SATURATION      50
 
 #define ENCODER_QUALITY_RANGE     2
+#define ENCODER_QUALITY_RANGE_AVC     8
```

Comment:
Does the new encoder really support quality level from 1 to 8? I think HIGH(1)/Medium(2, default)/Low(3) is enough.